frontend/i18n/bs/docusaurus-plugin-content-docs/current/scripting/functions/tickcount.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: tickcount
 sidebar_label: tickcount
-description: Ova se funkcija može koristiti kao zamjena za GetTickCount, jer vraća broj milisekundi od pokretanja poslužitelja.
+description: Ova se funkcija vraća broj milisekundi od pokretanja poslužitelja.
```

Comment:
Just remove "se" from here, let it be like this: Ova funkcija vraća broj milisekundi od pokretanja poslužitelja.